addons/mail/static/src/new/public/discuss_public.xml

```diff
@@ -2,9 +2,12 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.discuss_public" owl="1">
-    <div>
-        DISCUSS PUBLIC
-    </div>
+    <div class="o_DiscussPublicView d-flex flex-column h-100 w-100 position-absolute" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
Unused `className` and `root` and `position-absolute`.
Make sure to not reuse old code that is useless. Only implement what is needed for the current working of the code.